docs/trace/links-based-sampler/LinksAndParentBasedSampler.cs

```diff
@@ -0,0 +1,53 @@
+// <copyright file="LinksAndParentBasedSampler.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using OpenTelemetry.Trace;
+
+namespace LinksAndParentBasedSamplerExample;
+
+/// <summary>
+///  An example of a composite sampler that has:
+///  1. A probabilistic sampler (ParentBased Sampler).
```

Comment:
Also just a small nit, but a parent-based sampler doesn't necessarily need to be probabilistic (as this depends on which delegate sampler is used).